codegen.yml

```diff
@@ -0,0 +1,7 @@
+schema: ./src/core/server/graph/schema/schema.graphql
+generates:
+  introspection.json:
```

Comment:
I would suggest we store the `introspection.json` file instead:

```
src/core/server/graph/schema/__generated__/introspection.json
```

The introspection is then tied closer to the schema available in that folder.